### ISSUE TITLE
Do not show the closing part of the animation, only the opening.

### ISSUE THREAD

**Issue by dxwil** [shubhamsinghshubham777/easy_animated_indexed_stack#2]
I want to create an animation where when I click onto another page, the current page instantly dissapears bu the next page slides up. I can't seem to figure it out as the animation value in teh builder goes from 1 to 0 back to 1. The 1 to 0 part is the closing animation which I can't figure out how to remove.

Thanks

**Comment by shubhamsinghshubham777** [comment]
I understand what you're trying to achieve, but unfortunately this it might not be possible with this library. Basically this library is nothing but an AnimationController where it simply:

1. Starts the animation
2. Updates the index on animation end
3. Reverses the animation

So all animations that you write inside `transitionBuilder` will be run in reverse. So, if you're okay in writing animations that look nice from either direction, then this library is for you ✅

**Comment by shubhamsinghshubham777** [comment]
The [animations](https://pub.dev/packages/animations) package on pub.dev has come up with a widget named `PageTransitionSwitcher` that can help you achieve your goal:

https://github.com/flutter/packages/blob/3515abab07d0bb2441277f43c2411c9b5e4ecf94/packages/animations/lib/src/page_transition_switcher.dart#L77-L81

**Comment by shubhamsinghshubham777** [comment]
Maybe in a new version, I can attempt implementing something similar, i.e. ask for 2 transition builders, one for the start animation and one for the reverse animation. That might allow you to disable any of the transitions as per your requirements. How does that sound?

**Comment by dxwil** [comment]
> `Caveat`: It doesn't provide the keeping-widgets-in-memory benefit like an IndexedStack

Yeah had a setup with a native animation but IndexedStack didn't work, so that's why I switched to your package. If you could implement this, that would be amazing. Please reply to this issue whenever you have time to implement this so I get notified :)

Thank you

**Comment by dxwil** [comment]
For context, I am trying to do this:


https://github.com/user-attachments/assets/5306fb7c-f752-484f-bd3e-df6eedeaf462

**Comment by shubhamsinghshubham777** [comment]
> For context, I am trying to do this:
> 
>  signal-2024-12-19-211422_002.mp4

Hi @dxwil! Apologies for responding so late. I'm not sure if it will be useful to you anymore, but I'll still be implementing what you asked for:

https://github.com/user-attachments/assets/bd42df73-7e7b-4b5d-977d-1e3933445154

In the upcoming version, I'll be adding two new parameters named `skipStart` and `skipEnd` which would default to `false` but if you only use `skipStart: true`, it should help you achieve your usecase.

**Comment by dxwil** [comment]
I'm unfortunately not using Flutter anymore, but that's still nice to hear. Others will definitely find this useful.

**Comment by shubhamsinghshubham777** [comment]
> I'm unfortunately not using Flutter anymore

Sad to hear this :( Still, wishing you all the best for your future endeavours :)

**Comment by shubhamsinghshubham777** [comment]
Added params `skipStart` and `skipEnd` to [version 0.1.0](https://pub.dev/packages/easy_animated_indexed_stack/changelog#010) ✅
